astroquery/logger.py

```diff
@@ -0,0 +1,39 @@
+import logging
```

Comment:
maybe move this file into `utils`? I think that would be nice to keep it out of the main namespace, and it would also keep it more consistent with sunpy. (on the other hand astropy keeps it in the top level, so there is precedence for both, I just have a slight preference to have fewer one-off files mixed with the modules)

Reply:
Indeed, I followed the `astropy` convention.